src/main/flight/mixer.c

```diff
@@ -814,6 +814,22 @@ float applyThrottleLimit(float throttle)
     return throttle;
 }
 
+void applyAirMode(float *motorMix, float motorMixMax)
+{
+    float absThrottle = ABS(throttle);
```

Comment:
is it possible for throttle to have a negative value??? if not then this is not needed

Reply:
Can't be negative on 3D mode?

Reply:
As not said... I remove the ABS

Reply:
I just looked and at line 652 of mixer.c constrains the throttle to be between 0 and 1.

Reply:
3d mode centers at 0.5 so it should be fine